### PR TITLE
User Notes Command

### PR DESCRIPTION
**Pull request by WillFJS**
This pull request is for the notes command requested by @rossloika #118

### CODE REVIEW

File: src/bot/commands/notes.js [hunk 1]
```diff
@@ -0,0 +1,607 @@
...
+                            name: '⏹️',
+                        },
+                    },
+                ],
+            },
+        ],
+    });
+
+    const sorted_user_notes = user_notes.sort((a, b) => b.record.epoch - a.record.epoch);
+    const user_notes_chunks = array_chunks(sorted_user_notes, 10);
```
Comment (MidSpike) [inline src/bot/commands/notes.js:439]:
```suggestion
    const user_notes_chunks = array_chunks(sorted_user_notes, 5);
```

**Review by TotoTheEgg — APPROVED** [review]

**Review by MidSpike — COMMENTED** [review]
@WillFJS everything seems to look correct at a glance.
Can you spring up a bot to test this branch?

**Comment by MidSpike** [comment]
@Inertia-Lighting/discord-bot-api-server please review

**Review by dr4wn — APPROVED** [review]
looks good !

**Review by MidSpike — APPROVED** [review]
Things are looking good,
@rossloika please review this pull request as it is a lot of code.

**Review by TotoTheEgg — APPROVED** [review]

**Review by rossloika — APPROVED** [review]

**Pull request merged**